.github/dependabot.yml

```diff
@@ -42,3 +48,118 @@ updates:
     labels:
       - "dependencies"
     open-pull-requests-limit: 5
+    target-branch: main
+
+  ##############
+  # stable/8.1 #
+  ##############
+  # Enable version updates for Java
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "03:00"
+      timezone: "UTC"
+    commit-message:
+      prefix: "deps(maven): "
+    labels:
+      - "dependencies"
+    open-pull-requests-limit: 5
+    target-branch: "stable/8.1"
+    ignore:
+      # Ignore major and minor updates, for stable branches we only want to get patches
+      - dependency-name: "*"
+        update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
```

Comment:
For every package-ecosystem these three lines are the only difference. For stable branches the ignore is setup to allow patch updates only.